blogposts/2020/announcing-sourcegraph-3.16.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "Sourcegraph 3.16: Version contexts, campaigns, and major syntax highlighting improvements"
+author: Christina Forney
+publishDate: 2020-05-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.16
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+
+Explore, navigate, and better understand all code, everywhere, faster with Sourcegraph Universal Code Search. Uber, Lyft, Yelp, and other enterprises rely on Sourcegraph to improve developer productivity and manage large-scale code changes.
+
+<div style="padding-left: 2rem">
+
+[**📌 Introducing version contexts to search past releases**](#introducing-version-contexts-to-search-past-releases)<br />
+
+[**📣 Smoother getting started experience for campaigns**](#smoother-getting-started-experience-for-campaigns)<br />
+
+[**🎨 Best in class syntax highlighting**](#best-in-class-syntax-highlighting)<br />
```

Comment:
"Best-in-class" - add hyphens because that is how I usually see it, and it also makes it easier to parse for non-native English speakers.